### PR TITLE
MINOR: Doc individual partition must fit on the server that host it

### PR DESCRIPTION
**Pull request by b1tfury**

### CODE REVIEW

**Comment by b1tfury** [comment]
@here Guys, this is a simple change, please correct me if I am wrong.

File: docs/introduction.html [hunk 1]
```diff
@@ -50,7 +50,7 @@
 <p>
 This combination of features means that Kafka consumers are very cheap&mdash;they can come and go without much impact on the cluster or on other consumers. For example, you can use our command line tools to "tail" the contents of any topic without changing what is consumed by any existing consumers.
 <p>
-The partitions in the log serve several purposes. First, they allow the log to scale beyond a size that will fit on a single server. Each individual partition must fit on the servers that host it, but a topic may have many partitions so it can handle an arbitrary amount of data. Second they act as the unit of parallelism&mdash;more on that in a bit.
+The partitions in the log serve several purposes. First, they allow the log to scale beyond a size that will fit on a single server. Each individual partition must fit on the server that host it, but a topic may have many partitions so it can handle an arbitrary amount of data. Second they act as the unit of parallelism&mdash;more on that in a bit.
```
Comment (guozhangwang) [inline docs/introduction.html]: `on the server that hosts it`?

**Comment by guozhangwang** [comment]
LGTM except one minor comment. cc @ijuma @gwenshap .

**Comment by ijuma** [comment]
Same as what @guozhangwang said.

File: docs/introduction.html (hunk 1 above)
Comment (b1tfury) [inline docs/introduction.html]: Yes, on the server that  host it.
If all looks good, then let's merge this.

Comment (guozhangwang) [inline docs/introduction.html]: @bitfurry `host it` -> `hosts it`.

Comment (b1tfury) [inline docs/introduction.html]: @guozhangwang thanks, it is done.

**Comment by b1tfury** [comment]
@guozhangwang @ijuma let's merge this, or we waiting for some more discussion ?

**Comment by guozhangwang** [comment]
Merged to trunk.

**Pull request closed without merging**